packages/ddex/publisher/src/services/publisherService.ts

```diff
@@ -42,9 +42,9 @@ const formatTrackMetadata = (
     copyrightLine: metadata.copyright_line,
     producerCopyrightLine: metadata.producer_copyright_line,
     parentalWarningType: metadata.parental_warning_type,
-    isStreamGated: metadata.is_stream_gated,
```

Comment:
nit: can just do `{ type: Boolean, default: false }` in the metadata in `pendingReleases.ts` so you don't have to do this `?? undefined` here

Reply:
changed